__tests__/app.test.js

```diff
@@ -38,4 +38,46 @@ describe("GET api/topics", () => {
         .get('/api/invalid')
         .expect(404)
     })
+})
+
+
+/* 
+SHOULD:
+be available on /api
+provide a description of all other endpoints
+
+RESPONDS WITH:
+an object describing all available endpoints on the API
+
+EACH ENDPOINT SHOULD INCLUDE:
+
+a brief description of the purpose and functionality of the endpoint.
+which queries are accepted.
+what format the request body needs to adhere to.
+what an example response looks like.
+*/
```

Comment:
I would remove the above notes once the test is written as the test describes this